app/components/dashboard/hypertension/downloads_dropdown_component.rb

```diff
@@ -0,0 +1,52 @@
+class Dashboard::Hypertension::DownloadsDropdownComponent < ApplicationComponent
+  include QuarterHelper
+
+  attr_reader :region, :report_scope, :period, :current_admin
+
+  def initialize(region:, report_scope:, period:, current_admin:)
+    @region = region
+    @period = period
+    @report_scope = report_scope
+    @current_admin = current_admin
+  end
+
+  def region_download_path
+    reports_region_download_path(@region, report_scope: report_scope, period: :quarter, format: :csv)
```

Comment:
[Style] As that from [here](https://github.com/simpledotorg/simple-server/pull/4114/files#diff-cd1f09bfbd003f015175838bd04308ce20e234e5f52e0d17c72b709a7a132ca0R15 )